atum/src/main/scala/za/co/absa/atum/utils/controlmeasure/ControlMeasureBuilder.scala

```diff
@@ -43,6 +49,26 @@ trait ControlMeasureBuilder {
 
 
 object ControlMeasureBuilder {
+
+  sealed trait ControlTypeStrategy
+  object ControlTypeStrategy {
+
+    private[controlmeasure] case class ControlTypeMapping(columnName: String, strategy: ControlTypeStrategy = Default)
```

Comment:
I have been considering exposing `case class ControlTypeMapping` and `CMB.def withAggregateColumns(controlTypeMappings: Seq[(String, ControlType)])` in order for the user to be able to have the option to define all specific types for columns at-once.

After some thought, I have decided not to do it now (as the same result can be obtained by assembling the mappings iteratively (`CMB.withAggregateColumn("colA", Specific(someControlType1)).withAggregateColumn("colB", Specific(someControlType2))`). We can expose it if needed, but it would be hard to unexposed.

Let's see what you, the reviewers, think about it.